### PR TITLE
feat(Alert): Added recommended conditions

### PR DESCRIPTION
**Pull request by nbaenam**
Added recommended conditions for mobile and browser.

This is the [ticket (NR-110458)](https://issues.newrelic.com/browse/NR-110458).

### CODE REVIEW

File: src/content/docs/browser/browser-monitoring/getting-started/browser-summary-page.mdx [hunk 1]
```diff
@@ -15,24 +15,31 @@ redirects:
 
 import browserBrowserSummary from 'images/browser_screenshot-full_browser-summary.webp'
 
+import browserAlertConditions from 'images/browser_screenshot-full_browser-alert-conditions.webp'
+
+import browserAddAlert from 'images/browser_screenshot-crop_browser-add-alert.webp'
+
+
```
Comment (ally-sassman) [inline src/content/docs/browser/browser-monitoring/getting-started/browser-summary-page.mdx:22]:
```suggestion
```

File: src/content/docs/browser/browser-monitoring/getting-started/browser-summary-page.mdx [hunk 2]
```diff
@@ -71,3 +78,36 @@ Performance problems may be related to the browser, or they may be related to so
...
+      title="Browser monitoring recommended alert conditions"
+      alt="Browser monitoring recommended alert conditions"
+      src={browserAddAlert}
+    />
+
+4. Click **Next**.
+
+5. Select a policy to get notified. You can select an existing one or [create a new one](/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/create-edit-or-find-alert-policy/#alert-policy-name).
+
+6. Click **Activate conditions**.
```
Comment (ally-sassman) [inline src/content/docs/browser/browser-monitoring/getting-started/browser-summary-page.mdx:113]:
```suggestion
1. Open the browser monitoring **Summary** page.
2. Click **Add recommended conditions**.

    <img
      width="80%;"
      style={{align: "left"}}
      title="Browser monitoring summary"
      alt="Screenshot of the New Relic browser monitoring Summary page"
      src={browserAlertConditions}
    />
    
3. Choose a number of recommended alert conditions.

    <img
      width="70%;"
      style={{align: "left"}}
      title="Browser monitoring recommended alert conditions"
      alt="Browser monitoring recommended alert conditions"
      src={browserAddAlert}
    />
    
4. Click **Next**.
5. Select a policy to get notified. You can select an existing one or [create a new one](/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/create-edit-or-find-alert-policy/#alert-policy-name).
6. Click **Activate conditions**.
```

File: src/content/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/mobile-apps-summary-page.mdx [hunk 3]
```diff
@@ -98,17 +103,17 @@ The Summary page captures five key app metrics:
 
 To view the Summary page for your mobile apps:
 
-1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Mobile > (select an app)**.
+1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > All Capabilities > Mobile > (select an app)**.
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/mobile-apps-summary-page.mdx:106]:
```suggestion
1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Mobile > (select an app)**.
```

File: src/content/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/mobile-apps-summary-page.mdx [hunk 4]
```diff
@@ -98,17 +103,17 @@ The Summary page captures five key app metrics:
...
-  title="new-mobile-apps-overview"
-  alt="new-mobile-apps-overview"
+  title="Mobile summary"
+  alt="Screenshot of the New Relic mobile summary page"
   src={mobileMobileSummary}
 />
 
 <figcaption>
-  **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Mobile > (select an app):** The Summary page provides charts and tables that you can drill down into, to gain insights about your mobile application's performance.
+  **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > All Capabilities > Mobile > (select an app):** The Summary page provides charts and tables that you can drill down into, to gain insights about your mobile application's performance.
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/mobile-apps-summary-page.mdx:116]:
```suggestion
  **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Mobile > (select an app):** The Summary page provides charts and tables that you can drill down into, to gain insights about your mobile application's performance.
```

**Review by ally-sassman — DISMISSED** [review]

File: src/content/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/mobile-apps-summary-page.mdx (hunk 3 above)
Comment (nbaenam) [reply src/content/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/mobile-apps-summary-page.mdx:106]: Good catch!

**Comment by nbaenam** [comment]
Triggered by a manual rebuild. This is [the build](https://build-e04206cf-02a5-4274-8320-11735390ecf4.gatsbyjs.io/).

**Pull request merged**